src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -60,6 +60,8 @@ internal LuceneSearchUtility(DynamoModel model)
         /// </summary>
         internal void InitializeLuceneConfig(string dirName, LuceneStorage storageType = LuceneStorage.FILE_SYSTEM)
         {
+            if (DynamoModel.IsHeadless) return;
```

Comment:
perhaps you should mention this change in the comments for IsHeadLess mode, or do you not intend to port this change to master?

Reply:
I can update comments there